### DISCUSSION TITLE
createMany() with "INSERT IGNORE"

### DISCUSSION THREAD

**Discussion by Be-P** [prisma/prisma#15716] [Q&A] [answered]
Hi guys, thanks for the great tool and support!

I would like to bulk-insert with `createMany()` many rows, and ignore violated unique constraints.
What is the best way to go? Does prisma support such feateure?

**Comment by pantharshit00** [comment, marked as answer]
Hello,

This is currently not possible. You will need to use raw query here if you want to use `INSERT IGNORE`. I would suggest you to open a feature request here about this: https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=

**Reply by vladi-stevanovic** [reply]
👋 Hello @Be-P did you perhaps have a chance to check Harshit's recommendation? 

Let us know if you still have any questions, or feel free to "Mark as answer" if this discussion is now solved! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
